src/messaging/ExchangeContext.cpp

```diff
@@ -266,6 +267,27 @@ CHIP_ERROR ExchangeContext::SendMessage(Protocols::Id protocolId, uint8_t msgTyp
     }
 }
 
+void ExchangeContext::SendStatusReport(const Protocols::SecureChannel::StatusReport & statusReport)
+{
+    ChipLogDetail(ExchangeManager, "Sending status report. Protocol code %u, exchange %u", statusReport.GetProtocolCode(),
+                  GetExchangeId());
```

Comment:
```suggestion
    ChipLogProgress(ExchangeManager, "Sending status report. Protocol code %u, exchange %" ChipLogFormatExchange, statusReport.GetProtocolCode(),
                    ChipLogValueExchange(this));
```

Exchange ids are not unique; this will log the id _and the direction_.